components/SearchSandbox/components/RSWrapper.js

```diff
@@ -80,14 +80,17 @@ export default class RSWrapper extends Component {
 		});
 	}
 
-	getAvailableDataField = (fieldType) => {
-		const { types } = propsMap[this.props.component][fieldType];
+	getAvailableDataField = () => {
+		let types = propsMap[this.props.component].dataField.types;
 
-		if (this.props.id === 'search' && fieldType !== 'categoryField') {
-			return Object.keys(this.props.mappings).filter(field =>
-				types.includes(this.props.mappings[field].type));
+		if(this.props.component === 'CategorySearch'){
```

Comment:
Why are we returning different fields for CategorySearch and DataSearch? They both share the same dataFields.

Reply:
Yeah they both have share the same datafield, and the category field is different (same as that of MultiList).

See the screenshot of the movies app: 

<img width="864" alt="screen shot 2018-09-28 at 5 11 24 pm" src="https://user-images.githubusercontent.com/22376783/46206445-9e3eb380-c341-11e8-9704-12aa8a29faac.png">

Reply:
Create a separate method for getting `categoryField`.

`dataField` should be isolated from `categoryField` utility.